crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -1012,4 +1055,29 @@ pub mod tests {
         assert_eq!(req_metrics.output_nodes.get_sample_sum(), 2. + 4.);
         assert_eq!(req_metrics.query_depth.get_sample_sum(), 1. + 3.);
     }
+
+    pub async fn test_health_check_impl() {
+        let server_builder = prep_schema(None, None);
+        let url = format!(
+            "http://{}:{}/health",
+            server_builder.state.connection.host, server_builder.state.connection.port
+        );
+        server_builder.build_schema();
+
+        let resp = reqwest::get(&url).await.unwrap();
+        assert_eq!(resp.status(), StatusCode::OK);
+
+        let url_with_param = format!("{}?max_checkpoint_ts_lag=10", url);
```

Comment:
I'm a *little* bit concerned that this method of testing will be flaky -- maybe just set the param to 1ms to make it vanishingly unlikely?

Reply:
Fixed!